### PR TITLE
Week 7 - assignment

### PR DESCRIPTION
**Pull request by Kavya-Gowtham**
Week-7 assignment

### CODE REVIEW

File: Week-7/Assignment-7.6/Queue.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    q.enqueue(2);
+    q.enqueue(1);
+    q.enqueue(3);
+    q.enqueue(2);
+    q.enqueue(1);
+    q.enqueue(4);
+    q.enqueue(2);
+    
+    document.write(q.dequeue());
+    document.write(q.dequeue());
```
Comment (TanishShah37) [inline Week-7/Assignment-7.6/Queue.js]: 7.6 Why we are printing Twice ?

File: Week-7/Assignment-7.5/nextGreaterElement.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	let s = [];
+	s.push(arr[0]);
+	for (var i = 1; i < n; i++)				  //Time Complexity = O(n)
+	{
+        if (s.length == 0) {
+			s.push(arr[i]);					    //Space Complexity = O(n)
+			continue;
+		}
+
+		while (s.length ==0 == false
```
Comment (TanishShah37) [inline Week-7/Assignment-7.5/nextGreaterElement.js]: 7.5 what is s.length ===0 == false ?
7.6 Why we are printing Twice ?

File: Week-7/Assignment-7.3/loop.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  }
+}
+
+push(10);
+push(20);
+push(30);
+push(40);
+push(50);
+
+//creating a loop here
```
Comment (TanishShah37) [inline Week-7/Assignment-7.3/loop.js:36]: 7.3 how will handle the case when head.next.next.next.next  fails as there can null referencing

**Review by TanishShah37 — CHANGES_REQUESTED** [review]
For 7.1 and 7.2 can have a common linkedlist function for inserting data into linkedlists
7.3 how will handle the case when head.next.next.next.next  fails as there can null referencing
7.5 what is s.length ===0 == false ?
7.6 Why we are printing Twice ?

File: Week-7/Assignment-7.3/loop.js (hunk 3 above)
Comment (Kavya-Gowtham) [reply Week-7/Assignment-7.3/loop.js:36]: I have made some changes for 7.5 and 7.6 please verify once

**Review by TanishShah37 — APPROVED** [review]

**Pull request merged**